### DISCUSSION TITLE
TWGL Discord?

### DISCUSSION THREAD

**Discussion by jyoung4242** [greggman/twgl.js#230] [Ideas]
Has there been any consideration for creating a Discord server for TWGL?

I just started learning TWGL and kinda looked for one, i think it would be super helpful...

**Comment by redblobgames** [comment]
Building a community is hard. I think the closest I've seen is the pixijs discord server, which has channels for webgl, webgpu, regl, babylon, threejs, etc. Maybe they could be persuaded to add a twgl channel too.

**Reply by jyoung4242** [reply]
A) you couldn't be more accurate, I'm a mod for a game dev discord, and understand entirely
and
B) Love your content @redblobgames, we're all big fans

**Comment by greggman** [comment]
Sorry there is no discord. If you have any questions you can ask here and I can try to answer.

I think twgl, being very low-level, would probably have a much smaller community than full on 2d/3d engines like pixi.js/three.js/babylon.js
